plasma_framework/python_tests/tests/contracts/root_chain/test_start_standard_exit.py

```diff
@@ -130,60 +130,6 @@ def test_start_standard_exit_from_deposit_must_be_exitable_in_minimal_finalizati
     assert testlang.get_standard_exit(deposit_id) == [NULL_ADDRESS_HEX, 0, 0, False]
 
 
-@pytest.mark.skip("Skipped due to mix with IFE")
-@pytest.mark.parametrize("num_outputs", [1, 2, 3, 4])
-def test_start_standard_exit_on_piggyback_in_flight_exit_valid_output_owner_should_fail(testlang, num_outputs):
```

Comment:
I would rather move this test than remove it.
I think we should have covered all interaction scenarios from an old doc: [https://github.com/omisego/elixir-omg/blob/v0.2/docs/standard_vs_in_flight_exits_interaction.md](https://github.com/omisego/elixir-omg/blob/v0.2/docs/standard_vs_in_flight_exits_interaction.md). 

Even if we have moved to ALD, the cross-spend scenarios still exist, even though they may now be handled differently - they are protocol-level.

Reply:
This is no longer a valid scenario. The one thing we should test here is that finalizing standard exit from the same output as ife exit does not exit the output twice. It's covered in the test added to process exits tests

Reply:
I think that the scenario 9 per se is still valid - `Mallory submits IFE -> Mallory piggybacks output -> Mallory does SE on output`. It should be now tested in some other way.

Reply:
feel like case 7 and 9 become same cases as there is no "blocking piggyback" part anymore?

Reply:
@pik694 scenario is valid except for the result, now it's ok to start an exit. It just won't get paid as ife is processed earlier.